airflow/providers/microsoft/azure/operators/asb.py

```diff
@@ -15,7 +15,9 @@
 # specific language governing permissions and limitations
 # under the License.
 import datetime
-from typing import TYPE_CHECKING, List, Optional, Sequence, Union
+from typing import TYPE_CHECKING, Any, List, Optional, Sequence, Union
+
+from azure.servicebus.management._models import AuthorizationRule
```

Comment:
Please move this to inside the `if TYPE_CHECKING` block

Reply:
Why I wonder? We don't do that elsewhere? Is there a particular reason why you want to have it in TYPE_CHECKING?

Reply:
Because it's sole use is in a type annotation, and in this particular case, it feels like a "private" module (the underscore prefix)

Reply:
moved the code inside `if TYPE_CHECKING`